File: Programmers/Lv_1/수박수박수박수박수박수?.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

string solution(int n) {
    string answer = "";

    for (int i = 1; i <= n; i++) {
        if (i % 2 == 1) answer += "수";
        else answer += "박";
    }

    return answer;
}

int main() {
    ios_base::sync_with_stdio(0);
    cin.tie(0);

    int n;
    cin >> n;

    cout << solution(n);

    return 0;
}

File: Programmers/Lv_1/수박수박수박수박수박수?.py
def solution(n):
    answer = ''
    for i in range(n):
        if i % 2 == 1:
            answer += '박'
        else:
            answer += '수'
    return answer

def solution2(n):
    str = "수박" * n
    
    return str[:n]

def solution3(n):
    return "수박"*(n//2) + "수"*(n%2)